### PR TITLE
[PROGRAMMES-6820] Adjusting image captions

### PR DESCRIPTION
**Pull request by cisco87**
https://sandbox.bbc.co.uk/programmes/articles/4RDMM7wNV5XyMkGB3Hvzy64/bbcsitetest1
http://pal.sandbox.dev.bbc.co.uk/programmes/articles/4RDMM7wNV5XyMkGB3Hvzy64/bbcsitetest1
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6820

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by benolot** [comment]
@cisco87 please double check this in browser, padding looks wrong. 
screen shot 2018-12-20 at 15 05 11

compared to what it looks like on live:

screen shot 2018-12-20 at 15 05 46

File: src/Ds2013/Presenters/Domain/ContentBlock/Image/image.html.twig [hunk 1]
```diff
@@ -8,5 +12,7 @@
         { 0: 1, 770: 1/2, 1008: '490px' },
         { 'is_lazy_loaded': false, 'ratio': 'auto' }
     ) -}}
-    <div class="centi island--vertical">{{ content_block.getBlock().getCaption() }}</div>
+    {% if content_block.getBlock().getCaption() is not empty %}
```
Comment (BjornTwachtmann) [inline src/Ds2013/Presenters/Domain/ContentBlock/Image/image.html.twig]: Does this duplicate https://github.com/bbc/programmes-frontend/pull/910 . If so can you have a word with Tom and decide which one to merge?

Comment (cisco87) [reply src/Ds2013/Presenters/Domain/ContentBlock/Image/image.html.twig]: That one has already been approved so I guess it makes sense to merge that one then I will take care of the conflicts here

Comment (cisco87) [reply src/Ds2013/Presenters/Domain/ContentBlock/Image/image.html.twig]: No wait, I think this one also takes care of the padding around the image

**Pull request merged**